chain/chain/Cargo.toml

```diff
@@ -40,7 +40,7 @@ near-logger-utils = {path = "../../test-utils/logger"}
 byzantine_asserts = []
 expensive_tests = []
 test_features = []
-delay_detector = ["delay-detector"]
+delay_detector = ["delay-detector/delay_detector"]
```

Comment:
as a follow up, we might think about removing all these forwarding Cargo.features. Rather than having 

```toml
[features]
delay_detector = ["delay-detector/delay_detector"]
```

in every intermediate Cargo.toml, we now can have just one at the top level in `neard/Cargo.toml`, as that would enable delay detection globally.

Reply:
@matklad Yes, that's a good idea. I'll have to think about it. Let's move the discussion to a separate PR.